project/ScalafixBuild.scala

```diff
@@ -266,7 +266,10 @@ object ScalafixBuild extends AutoPlugin with GhpagesKeys {
     )
   )
 
-  private val PreviousScalaVersion = Map[String, String]()
+  private val PreviousScalaVersion = Map(
+    "2.12.13" -> "2.12.12",
+    "2.13.4" -> "2.13.3"
```

Comment:
you didn't actually bump 2.13 in this PR, so you should remove this line as it tries to get the previous artifact with 2.13.3 while it was already built for 2.13.4

Reply:
Ach, my bad! Somehow, I thought it wan't yet updated to 2.13.4.